app/java-2/src/main/java/com/google/SquirrelCensusDictionaryBuilder.java

```diff
@@ -10,18 +10,32 @@
 import java.util.Hashtable;
 
 class SquirrelCensusDictionaryBuildResult {
-    public Dictionary<String, Integer> dictionary;
+    public Dictionary<String, SquirrelSegment> dictionary;
     public int numOfRowsProcessed;
     public int numOfRowsIgnored;
 }
 
+class SquirrelSegment {
+    
+    // Variables names may not follow Java conventions because these names need
+    // to match names used for the Python and JavaScript version of this code.
```

Comment:
Strictly speaking, these values match the data source, not the other languages (I prefer not to have one implementation mention another implementation by name, especially since the languages may change over time)

Reply:
Good point about `Running`, `Chasing`, etc. being derivatives of the CSV (raw) data file.
But I was also referring to `_count` and the convention to prefix with `_` for private variables.

That said, I will replace `Python and JavaScript` with `other languages` and add a point about deriving names from the CSV (raw).